examples/source/user-consent/Client_Side_User_Consent_Flow.html

```diff
@@ -1,5 +1,7 @@
 <!---
-
+  
+experiments:
+  - amp-consent-geo-override
 preview: default
 skipValidation: 'true'
 teaserImage: '/static/samples/img/teaser/basic_user_consent_flow.jpg'
```

Comment:
@sebastianbenz what is the teaserImage? We'll probably need to rename the image as well.

Reply:
These images are screen shots of promptUis. I can't find where they are getting used in the website, but IMO they don't need to be updated.